src/query_parser/v2/parser.y

```diff
@@ -944,13 +966,66 @@ vector_score_field(A) ::= as STOPWORD(B). {
   A.type = QT_TERM;
 }
 
-// Every vector query will have basic command part. Right now we only have KNN command.
+// Use query attributes syntax
+query ::= expr(A) ARROW LSQB vector_query(B) RSQB ARROW LB attribute_list(C) RB. {
```

Comment:
Why do we need this? `ATTRIBUTE` should take care of this.

Reply:
Not sure what you mean... Note that this rule allows reducing the vector query + query attributes section into a whole query. Without this rule we cannot parse queries such as: `@t:text=>[KNN 10 @v $blob]=>{$ef_runtime:100}`

Reply:
> Not sure what you mean... Note that this rule allows reducing the vector query + query attributes section into a whole query. Without this rule we cannot parse queries such as: `@t:text=>[KNN 10 @v $blob]=>{$ef_runtime:100}`

Then, we need to fix the rule.